multichain-testing/package.json

```diff
@@ -8,7 +8,8 @@
     "lint:eslint": "eslint .",
     "lint:types": "tsc --noEmit",
     "lint-fix": "yarn lint:eslint --fix",
-    "test": "ava",
+    "test": "ava --config ava.main.config.js",
```

Comment:
`test` could test both configs. or it could echo instructions on how to test, since it's a long slow test